From b86936a58a9d20945c6bca62a816f24f8a4d15b3 Mon Sep 17 00:00:00 2001
From: Jouni Malinen <j@w1.fi>
Date: Fri, 22 Dec 2023 14:26:34 +0200
Subject: [PATCH] tests: Use longer timeout with sigma_dut in DPP
 reconfiguration steps

Some of these can take close to the previously used timeouts and this
could result in reporting failures in cases that worked fine.

Signed-off-by: Jouni Malinen <j@w1.fi>
---
 tests/hwsim/test_sigma_dut.py | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/tests/hwsim/test_sigma_dut.py b/tests/hwsim/test_sigma_dut.py
index 973a13f5f..cb289099f 100644
--- a/tests/hwsim/test_sigma_dut.py
+++ b/tests/hwsim/test_sigma_dut.py
@@ -4084,7 +4084,7 @@ def test_sigma_dut_dpp_reconfig_enrollee(dev, apdev):
             raise Exception("Failed to start listen operation")
         dev[1].dump_monitor()
 
-        res = sigma_dut_cmd("dev_exec_action,program,DPP,DPPActionType,DPPReconfigure,DPPTimeout,6,DPPWaitForConnect,Yes", timeout=10)
+        res = sigma_dut_cmd("dev_exec_action,program,DPP,DPPActionType,DPPReconfigure,DPPTimeout,16,DPPWaitForConnect,Yes", timeout=20)
         if "status,COMPLETE,ReconfigAuthResult,OK,ConfResult,OK,NetworkConnectResult,OK" not in res:
             raise Exception("Unexpected reconfiguration result: " + res)
 
@@ -4106,7 +4106,7 @@ def test_sigma_dut_dpp_reconfig_enrollee(dev, apdev):
         dev[0].dump_monitor()
         dev[1].dump_monitor()
 
-        res = sigma_dut_cmd("dev_exec_action,program,DPP,DPPActionType,DPPReconfigure,DPPTimeout,6,DPPWaitForConnect,Yes", timeout=30)
+        res = sigma_dut_cmd("dev_exec_action,program,DPP,DPPActionType,DPPReconfigure,DPPTimeout,16,DPPWaitForConnect,Yes", timeout=30)
         if "status,COMPLETE,ReconfigAuthResult,OK,ConfResult,OK,NetworkConnectResult,OK" not in res:
             raise Exception("Unexpected reconfiguration [2] result: " + res)
 
@@ -4165,7 +4165,7 @@ def test_sigma_dut_dpp_reconfig_enrollee_sae(dev, apdev):
             raise Exception("Failed to start listen operation")
         dev[1].dump_monitor()
 
-        res = sigma_dut_cmd("dev_exec_action,program,DPP,DPPActionType,DPPReconfigure,DPPTimeout,6,DPPWaitForConnect,Yes", timeout=10)
+        res = sigma_dut_cmd("dev_exec_action,program,DPP,DPPActionType,DPPReconfigure,DPPTimeout,16,DPPWaitForConnect,Yes", timeout=20)
         if "status,COMPLETE,ReconfigAuthResult,OK,ConfResult,OK,NetworkConnectResult,OK" not in res:
             raise Exception("Unexpected reconfiguration result: " + res)
 
@@ -4187,7 +4187,7 @@ def test_sigma_dut_dpp_reconfig_enrollee_sae(dev, apdev):
         dev[0].dump_monitor()
         dev[1].dump_monitor()
 
-        res = sigma_dut_cmd("dev_exec_action,program,DPP,DPPActionType,DPPReconfigure,DPPTimeout,6,DPPWaitForConnect,Yes", timeout=30)
+        res = sigma_dut_cmd("dev_exec_action,program,DPP,DPPActionType,DPPReconfigure,DPPTimeout,16,DPPWaitForConnect,Yes", timeout=30)
         if "status,COMPLETE,ReconfigAuthResult,OK,ConfResult,OK,NetworkConnectResult,OK" not in res:
             raise Exception("Unexpected reconfiguration [2] result: " + res)
 
@@ -4239,7 +4239,7 @@ def test_sigma_dut_dpp_reconfig_configurator(dev, apdev):
         if "OK" not in dev[1].request(cmd):
             raise Exception("Failed to start reconfiguration")
 
-        res = sigma_dut_cmd("dev_exec_action,program,DPP,DPPActionType,DPPReconfigure,DPPProvisioningRole,Configurator,DPPConfEnrolleeRole,STA,DPPSigningKeyECC,P-256,DPPConfIndex,2,DPPListenChannel,6,DPPTimeout,6", timeout=10)
+        res = sigma_dut_cmd("dev_exec_action,program,DPP,DPPActionType,DPPReconfigure,DPPProvisioningRole,Configurator,DPPConfEnrolleeRole,STA,DPPSigningKeyECC,P-256,DPPConfIndex,2,DPPListenChannel,6,DPPTimeout,16", timeout=20)
         if "status,COMPLETE,ReconfigAuthResult,OK,ConfResult,OK" not in res:
             raise Exception("Unexpected reconfiguration result: " + res)
 
@@ -4297,7 +4297,7 @@ def run_sigma_dut_dpp_reconfig_proto(dev, apdev, dpp_step):
         if "OK" not in dev[1].request(cmd):
             raise Exception("Failed to start reconfiguration")
 
-        res = sigma_dut_cmd("dev_exec_action,program,DPP,DPPActionType,DPPReconfigure,DPPProvisioningRole,Configurator,DPPConfEnrolleeRole,STA,DPPSigningKeyECC,P-256,DPPConfIndex,2,DPPStep,%s,DPPFrameType,ReconfigAuthRequest,DPPIEAttribute,ProtocolVersion,DPPListenChannel,6,DPPTimeout,6" % dpp_step, timeout=10)
+        res = sigma_dut_cmd("dev_exec_action,program,DPP,DPPActionType,DPPReconfigure,DPPProvisioningRole,Configurator,DPPConfEnrolleeRole,STA,DPPSigningKeyECC,P-256,DPPConfIndex,2,DPPStep,%s,DPPFrameType,ReconfigAuthRequest,DPPIEAttribute,ProtocolVersion,DPPListenChannel,6,DPPTimeout,16" % dpp_step, timeout=20)
         if "status,COMPLETE,ReconfigAuthResult,Errorsent" not in res:
             raise Exception("Unexpected reconfiguration result: " + res)
 
-- 
2.43.0

